pkg/helm/presets/backend/high-resources.yaml

```diff
@@ -6,7 +6,7 @@ k8sWatcher:
       memory: 300Mi
     limits:
       cpu: 100m
```

Comment:
also increase the cpu limit to 500mCPU 🙏

Reply:
Fixed